README.md

```diff
@@ -204,6 +204,8 @@ The low-level API is as follows:
 
 For examples how to use them, please refer to the README of the above mentioned solc-js releases.
 
+**Note**: These low-level functions still present for compatibility reasons to support all compiler binaries, but they were superseded by the `compile()` function and are no longer required in newer versions. The functions `compileSingle`, `compileMulti` and `compileCallback` are always `null` in the newer versions.
```

Comment:
I'm no native English speaker, but using `present` here sounds somehow weird to me. Why not "These low-level functions remain available for compatibility reasons." IMHO a simpler sentence.

Reply:
I will change it :)